widget/view.controller.js

```diff
@@ -33,7 +35,9 @@
         $scope.toggle = [];
         $scope.toggleRemediation = true;
         $scope.toggleConnectorConfig = [];
```

Comment:
toggleConnectorConfig toggle keyword is for true/false 
please update the name

Reply:
its save true/false in list format